### PR TITLE
Add Borrow command

### PR DESCRIPTION
**Pull request by jelneo**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/BorrowCommand.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+public class BorrowCommand extends UndoableCommand {
+
+    public static final String COMMAND_WORD = "borrow";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": increase the debt of a person by "
+            + "the amount of money entered.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + "AMOUNT (must have at least 1 digit and either a positive integer or a positive number with "
+            + "two decimal places)\n"
+            + "Example: " + COMMAND_WORD + " 1 120.50";
```
Comment (jaivigneshvenugopal) [inline src/main/java/seedu/address/logic/commands/BorrowCommand.java:26]: You wanted a comma is it jel? Like "1,120.50" ?

**Review by jaivigneshvenugopal — APPROVED** [review]
Great work jel!

File: src/main/java/seedu/address/logic/commands/BorrowCommand.java (hunk 1 above)
Comment (jelneo) [reply src/main/java/seedu/address/logic/commands/BorrowCommand.java:26]: 1 is the index of the person who borrowed extra cash, followed by amount owed which is 120.50

File: src/main/java/seedu/address/model/AddressBook.java [hunk 2]
```diff
@@ -208,6 +215,35 @@ public void removeTag(Tag t) throws TagNotFoundException {
...
+        Name name = target.getName();
+        Phone phone = target.getPhone();
+        Email email = target.getEmail();
+        Address address = target.getAddress();
+        PostalCode postalCode = target.getPostalCode();
+        Debt newDebt = target.getDebt();
+        newDebt.addToDebt(amount);
+        Deadline deadline = target.getDeadline();
+        Set<Tag> tags = target.getTags();
+        Person editedPerson = new Person(name, phone, email, address, postalCode, newDebt, deadline, tags);
```
Comment (lawwman) [inline src/main/java/seedu/address/model/AddressBook.java]: The DateBorrow field might be affected here. The dateBorrow field is created when a new Person is created. You have to set the date from the ReadOnlyPerson target.

Comment (jelneo) [reply src/main/java/seedu/address/model/AddressBook.java]: fixed

**Review by lawwman — APPROVED** [review]
Good Job!

**Review by khooroko — APPROVED** [review]
looks good aside from travis being irritating

**Pull request merged**